about.html

```diff
@@ -0,0 +1,150 @@
+        <section class="top-bg">
+            <section class="hero container">
+                <br>
+                <h4 class="greetings">"Explore Your Dream Career with Aspire"</h4>
+                <h2 class="hero-title">Get connected <br /> to the experts Globally</h2>
+                <br>
+            </section>
+
+            <section class="message">
+
+                <div class="container">
+                    <p class="message-p p-3 border border-white">
+                        At Career Guiders, we help young people explore different career options with the help of
+                        industry experts. Our mission is to reduce unemployment in Africa by funding talents and
+                        connecting young people with experts in their field of interest. Join us today and start your
+                        journey to success!
+                    </p>
+                    <p>please contact us per Email for any further questions.</p>
+                    <b class="underline">komezarsene15@gmail.com</b>
+                </div>
+            </section>
+            <br><br><br>
+        </section>
+        <section class="about-program bg-white">
+            <div class="container">
+                <br>
+                <h4 class="text-center section-title-black">Our Mission</h4>
+                <hr class="title-line">
+
+                At Career Guiders, we strive to reduce unemployment in Africa by helping young people discover and
+                pursue their interests and talents.
+                </p>
+                <br>
+                <div class="container"><img src="./images/aspire-logo.png" width="90%" height="100px" alt="Aspire logo">
+                </div>
+                <br>
+            </div>
+        </section>
+        <hr>
+
+        <section class="past-program bg-white">
+            <div class="container">
+                <br>
+                <h4 class="text-center section-title-black">Our Recent Talks</h4>
+                <hr class="title-line">
+
+                <p>Take a look at our past career guidance and feel free to leave a comment!.</p>
+
+                <div class="past-summit">
+                    <div class="summit-one text-center">
+                        <div class="summit-sub">
+                            <br><br>
+                            <h2 class="text-white">2020</h2>
+                            <p class="text-white">Rwanda: Kigali Yourth Celemon</p>
+                        </div>
+                    </div>
+
+                    <div class="summit-two text-center mt-3">
+                        <div class="summit-sub">
+                            <br><br>
+                            <h2 class="text-white">2023</h2>
+                            <p class="text-white">Kenya: Mount Kenya University</p>
+                        </div>
+                    </div>
+                </div>
+                <br><br>
+            </div>
+        </section>
```

Comment:
Your design
![image](https://github.com/Arsenethierry/capstone-project-aspire-web/assets/78251927/e58193a2-5233-475c-a53e-7124d105e349)

Guideline
![image](https://github.com/Arsenethierry/capstone-project-aspire-web/assets/78251927/dc567746-0c33-4aef-a951-dc69ce068ad8)

Please kindly adjust the spacing `margin/padding` of the contents of this section to match the given design as much as possible